app/src/main/java/com/discord/simpleast/sample/SampleTexts.kt

```diff
@@ -197,6 +197,23 @@ object SampleTexts {
     ```
   """
 
+  private const val CODE_BLOCK_JAVASCRIPT = """
+    JavaScript code block:
+    ```js
+    const { performance } = require('perf_hooks');
+    function getMem() {
+      return Object.entries(process.memoryUsage())
+               .map(([K, V]) => `\${K}: \${(V / (2 ** 1024)).toFixed(1)}MB`)
```

Comment:
```suggestion
               .map(([K, V]) => `${'$'}{K}: ${'$'}{(V / (2 ** 1024)).toFixed(1)}MB`)
```
Raw strings don't support escaping literal characters